Alerting: new min_interval_seconds option to enforce a minimum evaluation frequency (#21188)
grafana/grafana: docs/sources/alerting/rules.md

### Name and Evaluation interval

Here you can specify the name of the alert rule and how often the scheduler should evaluate the alert rule.

> Note: A minimum interval can be specified in the `alerting.min_interval_seconds` config field, to set a minimum time between evaluations.

Reviewer: ```suggestion
**Note:** You can set a minimum interval in the `alerting.min_interval_seconds` config field, to set a minimum time between evaluations.
```

Reviewer: Try to address instructions to "you" rather than writing them in the passive voice. They tend to be clearer and require fewer words.

Reviewer 2: Given my other comment would suggest linking to [configuration documentation](https://grafana.com/docs/grafana/latest/installation/configuration/#alerting) of this property.

Reviewer: Good point, @marefr . That would be a good improvement.
